--- origsrc/highlight-3.23/filetypes.conf
+++ src/highlight-3.23/filetypes.conf
@@ -43,7 +43,7 @@ FileMapping = {
  { Lang="progress", Extensions={"p", "i", "w"} },
  { Lang="ruby", Extensions={"rb","ruby", "pp", "rjs", "gemfile", "rakefile"} },
  { Lang="rexx", Extensions={"rex", "rx", "the"} },
- { Lang="sh", Extensions={"bash", "zsh", "ebuild", "eclass"} },
+ { Lang="sh", Extensions={"bash", "zsh", "ebuild", "eclass", "cygport", "cygpart", "cygclass"} },
  { Lang="smalltalk", Extensions={"st", "gst", "sq"} },
  { Lang="sybase", Extensions={"sp"} },
  { Lang="tcl", Extensions={"wish", "itcl"} },
