--- origsrc/highlight-3.13/src/gui-qt/highlight.pro	2013-01-18 16:43:59.000000000 -0600
+++ src/highlight-3.13/src/gui-qt/highlight.pro	2013-05-27 23:37:00.715290800 -0500
@@ -33,7 +33,7 @@ unix:LIBS += -L.. -lhighlight
 unix {
     LIBS += -llua5.1
     CONFIG += link_pkgconfig
-    PKGCONFIG += lua5.1
+    PKGCONFIG += lua
 }
 
 win32:QMAKE_POST_LINK = f:/upx/upx.exe --best ../../highlight-gui.exe
